.github/workflows/ci.yml

```diff
@@ -8,13 +8,14 @@ on:
 
 jobs:
   lint:
+    name: Lint
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
         with:
-          ref: ${{ github.head_ref }}
+          ref: ${{ github.ref }}
```

Comment:
Why use `ref` over `head_ref`?

Reply:
`head_ref` value is only available for the branch created in the main repo, not the forked one. That's why CI has been failed with the error when tried to fetch the repo. We should test auto-commit steps afterward.

Reply:
FYI: This is the relevant issue [Checkout V2](https://github.com/actions/checkout/issues/366)